playbooks/roles/install-epel/tasks/epel.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Install Extra Packages for Enterprise Linux (EPEL)
+  yum:
+    name:
+      - "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
+  become: true
```

Comment:
I believe this provisioner needs to run as root which probably renders the ubiquitous explicit priv esc unnecessary? But doesn't hurt, per docs default is always id0